packages/graphql-server/src/resolvers/office/api.ts

```diff
@@ -0,0 +1,138 @@
+import { fetcher, setQueryParams } from '@reapit/elements'
+import logger from '../../logger'
+import { ServerContext } from '../../app'
+import errors from '../../errors'
+import { API_VERSION, URLS } from '../../constants/api'
+import {
+  GetOfficesParams,
+  OfficeModel,
+  PagedResultOfficeModel_,
+  CreateOfficeModel,
+  UpdateOfficeParams,
+  GetOfficeParams,
+} from './offices'
+
+export const fetchOffices = (args: GetOfficesParams, context: ServerContext) =>
+  fetcher({
+    url: `${URLS.offices}/?${setQueryParams(args)}`,
+    api: process.env['PLATFORM_API_BASE_URL'],
+    method: 'GET',
+    headers: {
+      authorization: context.authorization,
+      'Content-Type': 'application/json',
+      'api-version': API_VERSION,
+    },
+  })
+
+export const fetchOfficeById = (args: GetOfficeParams, context: ServerContext) =>
+  fetcher({
+    url: `${URLS.offices}/${args.id}`,
+    api: process.env['PLATFORM_API_BASE_URL'],
+    method: 'GET',
+    headers: {
+      authorization: context.authorization,
+      'Content-Type': 'application/json',
+      'api-version': API_VERSION,
+    },
+  })
+
+export const fetchCreateOffice = (args: CreateOfficeModel, context: ServerContext) =>
+  fetcher({
+    url: `${URLS.offices}`,
+    api: process.env['PLATFORM_API_BASE_URL'],
+    method: 'POST',
+    headers: {
+      authorization: context.authorization,
+      'Content-Type': 'application/json',
+      'api-version': API_VERSION,
+    },
+    body: args,
+  })
+
+export const fetchUpdateOffice = (args: UpdateOfficeParams, context: ServerContext) => {
```

Comment:
it should `callUpdateOfficeAPI` because fetch and update also 2 verb in function name